### PR TITLE
#9 Implemented DenseNet121 for multi-label chest X-ray classification with TPU acceleration, data preprocessing, training, and evaluation.

### LINKED ISSUE #9: DenseNet Modelling

### PR DESCRIPTION
**Pull request by PranjalDhamane**
In this notebook I have:

1. Loaded, preprocessed and normalized chest x-ray images and their corresponding labels.
Detected and configured TPU (Tensor Processing Unit) for accelerated training.
2. Defined a DenseNet121 model for multi-label classification of chest X-ray images.
Set up data generators, callbacks (e.g., ModelCheckpoint, EarlyStopping), and compiled the model with appropriate loss functions and optimizers.
3. Trained the model using the preprocessed data and evaluated its performance.
4. Visualized the training process and results using plots.

### CODE REVIEW

File: python/notebook/chest-x-rays-densenet.ipynb
Comment (Asifdotexe) [inline python/notebook/chest-x-rays-densenet.ipynb:1]: Is this file needed for production or it is a duplicate?

**Review by tanvisivaraj — APPROVED** [review]

File: python/notebook/chest-x-rays-densenet.ipynb
Comment (PranjalDhamane) [reply python/notebook/chest-x-rays-densenet.ipynb:1]: It's needed for the production.

Comment (PranjalDhamane) [reply python/notebook/chest-x-rays-densenet.ipynb:1]: Since my laptop was not able to process such large data even using cuda I had to use kaggle's TPU for running the model, hence I have uploaded the whole file. And with lesser data the model's performance was lacking we came to the conclusion of using kaggle's TPU
